ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/Serialisation/Golden.hs

```diff
@@ -215,16 +215,23 @@ goldenTests testName examples enc goldenFolder mCDDL
   | otherwise =
       testGroup
         testName
-        [ goldenTestCBOR testName' example enc (goldenFolder </> testName') mCDDL
+        [ goldenTestCBOR testName' example enc (goldenFolder </> testName') mCDDL'
         | (mbLabel, example) <- examples
         , let testName' = case mbLabel of
                 Nothing -> testName
                 Just label -> testName <> "_" <> label
+              mCDDL' = skipCDDLCheckWithPerasCert mbLabel mCDDL
         ]
  where
   labels :: [Maybe String]
   labels = map fst examples
 
+  -- TODO this needs to be removed before closing:
+  -- https://github.com/tweag/cardano-peras/issues/103
+  skipCDDLCheckWithPerasCert mbLabel
+    | Just label <- mbLabel, "WithPerasCert" `isSuffixOf` label = const Nothing
+    | otherwise = id
+
```

Comment:
This is a workaround to avoid CDDL compliance tests when a ShelleyBlock includes a Peras certificate. It should be fixed after tweaking the CDDL spec accordingly. 

See also: https://github.com/orgs/tweag/projects/78/views/1?filterQuery=cddl&pane=issue&itemId=127206584&issue=tweag%7Ccardano-peras%7C103